composer.json

```diff
@@ -53,5 +53,8 @@
     },
     "bin": [
         "bin/doctrine-migrations"
-    ]
+    ],
+    "platform": {
+        "php": "7.1.x"
```

Comment:
@jwage Why exactly was this added?
a) it's invalid here, it must be in `"config"` node (has no effect here)
b) using `7.1.x` is invalid, it must be a specific version (i.e. `7.1.3`)
c) having this here will likely cause issues with SF4 since Migrations require `^7.1` and platform override therefore should be `7.1.0`

Reply:
It was suggested by @theofidry but i must have gotten it wrong. Can you record an issue and I will fix today.

Reply:
#701